lib/versioning/poetry/index.spec.ts

```diff
@@ -361,15 +361,15 @@ describe(getName(), () => {
           currentVersion: '5.0.0',
           newVersion: '5.1.7',
         })
-      ).toEqual('5');
+      ).toBeNull();
       expect(
         versionig.getNewValue({
           currentValue: '5',
           rangeStrategy: 'bump',
           currentVersion: '5.0.0',
           newVersion: '6.1.7',
         })
-      ).toEqual('6');
+      ).toBeNull();
```

Comment:
Why are these null now?

Reply:
I think poetry follows semver but doesn't enforce it. so i think it will do this `5.0.0` for `5` .
https://github.com/abn/poetry-core/blob/fbfcda73b2c5524c3eda29b1d9b70734a3a9dcf3/tests/semver/test_version.py#L13

This test files is maybe a good example for us.